### PR TITLE
[zh-cn]: update the translation of Document `createProcessingInstruction()` method

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/document/createprocessinginstruction/index.md [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
+title: Document：createProcessingInstruction() 方法
 slug: Web/API/Document/createProcessingInstruction
+l10n:
+  sourceCommit: acfe8c9f1f4145f77653a2bc64a9744b001358dc
 ---
 
 {{APIRef("DOM")}}
 
-`createProcessingInstruction()` 创建一个新的处理指令节点，并返回。
+`createProcessingInstruction()` 生成新的 [processing instruction](/zh-CN/docs/Web/API/ProcessingInstruction) 节点并返回。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/document/createprocessinginstruction/index.md:10]: Should `processing instruction` conecept been translated?

File: files/zh-cn/web/api/document/createprocessinginstruction/index.md [hunk 2]
```diff
@@ -1,44 +1,60 @@
...
+```js-nolint
+createProcessingInstruction(target, data)
 ```
 
-### Parameters
+### 参数
+
+- `piNode` 是生成的 {{ domxref("ProcessingInstruction") }} 节点。
+- `target` 是包含处理指令的第一部分字符串（即 `<?target … ?>`）。
+- `data` 是包含处理指令在目标之后应携带的任何信息字符串，数据由你决定，但不能包含 `?>`，因为这会关闭处理指令。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/document/createprocessinginstruction/index.md:24]: The concept has been translated here.

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Just have some confusions,  /cc @yin1999, thx.

File: files/zh-cn/web/api/document/createprocessinginstruction/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/api/document/createprocessinginstruction/index.md:10]: Yes

Comment (T34-active) [inline files/zh-cn/web/api/document/createprocessinginstruction/index.md:10]:
```suggestion
`createProcessingInstruction()` 生成新的[处理指令](/zh-CN/docs/Web/API/ProcessingInstruction)节点并返回。
```

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thx. @T34-active

**Pull request merged**